mne/time_frequency/multitaper.py

```diff
@@ -451,9 +451,10 @@ def _mt_spectra(x, dpss, sfreq, n_fft=None):
     return x_mt, freqs
 
 
-def _psd_multitaper(x, sfreq, fmin=0, fmax=np.inf, bandwidth=None,
-                    adaptive=False, low_bias=True, normalization='length',
-                    n_jobs=1):
+@verbose
+def psd_array_multitaper(x, sfreq, fmin=0, fmax=np.inf, bandwidth=None,
```

Comment:
fair enough.

> BTW - you mean the default of 1 or 1000?

1. seems more natural when nothing is specified? why should it be 1000?